[TT-10423] Update documentation for master (#3507)
TykTechnologies/tyk-docs: tyk-docs/content/shared/mdcb-config.md

Sets the batch size for mongo results.

### analytics.driver
EV: <b>TYK_MDCB_ANALYTICSCONFIG_DRIVER</b><br />
Type: `string`<br />

> Determines the driver used by the storage. It could be `mongo-go` to use the official mongo driver for go or `mgo` to use the old driver. By default, the value is `mgo`.

Reviewer: ```suggestion
Specifies which MongoDB driver should be used by the storage. Valid values are `mongo-go` to use the official mongo driver for Go or `mgo` to use the old driver. The default value is `mgo`.
```

### analytics.mongo_direct_connection
EV: <b>TYK_MDCB_ANALYTICSCONFIG_MONGODIRECTCONNECTION</b><br />
Type: `bool`<br />

MongoDirectConnection informs whether to establish connections only with the specified seed servers,

> or to obtain information for the whole cluster and establish connections with further servers too.

Reviewer: ```suggestion
or to discover and establish connections with further servers within the cluster.
```

If true, the client will only connect to the host provided in the ConnectionString
and won't attempt to discover other hosts in the cluster. Useful when network restrictions

> prevent discovery, such as with SSH tunneling. Default is false.

Reviewer: ```suggestion
restrictions prevent discovery, such as with SSH tunneling. Default is false.
```
